ui/app/AppLayouts/Profile/views/NotificationsView.qml

```diff
@@ -352,8 +355,10 @@ SettingsContentBase {
         }
 
         NotificationAppearancePreviewPanel {
-            Layout.preferredWidth: root.contentWidth
+            Layout.preferredWidth: root.contentWidth - Theme.padding * 2
```

Comment:
Why not fillWidth? Is it too large?

Reply:
I was trying to follow what is used there to gain proper behavior without refactoring the whole view, but it should be done at some point. But yeah, ideally it should be.

Reply:
> Why not fillWidth? Is it too large?

I think there's some extra margins/padding outside of this view; fillwidth will not work then